src/js/views/forms/form/form.scss

```diff
@@ -26,6 +26,7 @@
   flex: 1;
   flex-direction: column;
   min-width: 744px;
+  overflow: hidden;
```

Comment:
Not having this was causing a double scrollbar to show when the form page height is very short:

![Screenshot 2025-06-12 at 2 41 17 PM](https://github.com/user-attachments/assets/550b7721-7859-4190-ae09-2c30ab19606b)

Reply:
But the overall form layout doesn't need to scroll, since the actual form content scrolls independently.